### PR TITLE
Replaced cassandra-uuid with uuid

### PR DESCRIPTION
**Pull request by hknustwmf**
Bumped the version number to 0.9.3

Bug: T223690

### CODE REVIEW

**Comment by Pchelolo** [comment]
Test fails unrelated. Merging

File: package.json [hunk 1]
```diff
@@ -38,17 +38,17 @@
...
     "htcp-purge": "^0.3.0",
-    "hyperswitch": "^0.10.4",
+    "hyperswitch": "^0.12.4",
     "mediawiki-title": "^0.6.5",
     "murmur-32": "^0.1.0",
     "node-rdkafka": "2.3.1",
     "ratelimit.js": "^1.8.0",
     "redis": "^2.8.0",
-    "service-runner": "^2.6.17"
+    "service-runner": "^2.6.17",
```
Comment (d00rman) [inline package.json]: Unrelated, but please bump service-runner to v2.7.1

Comment (hknustwmf) [reply package.json]: Updated to 2.7.1

**Comment by d00rman** [comment]
Published as [v0.9.3](https://github.com/wikimedia/change-propagation/tree/v0.9.3)

**Pull request merged**